code/modules/mob/living/carbon/carbon.dm

```diff
@@ -365,16 +365,9 @@
 	to_chat(src, "<span class='notice'>You successfully [cuff_break ? "break" : "remove"] [I].</span>")
 
 	if(cuff_break)
+		. = !(I == (handcuffed || legcuffed))

```

Comment:
if `handcuffed` is not null and not `I` this will return false even if `legcuffed` is `I`

well true after it's been notted